app/models/log.js

```diff
@@ -21,7 +23,7 @@ var Request = Ember.Object.extend({
   },
 
   handle(body, status, xhr) {
-    if (config.pro) {
+    if (config.featureFlags.pro) {
```

Comment:
Unsure if it's needed now, but in the future we could make the `Request` object container aware, so we can use a service.